packages/indexer-generator/README.md

```diff
@@ -41,12 +41,12 @@ cd packages/<idl-name>
 npm i && npm run build
 ```
 3. Add your RPC on SOLANA_RPC env
-4. Grant execution permission to the script and run it
+4. Run he indexer by executing run script
```

Comment:
no chmod needed anymore? If you rebase on main, you see that I added a `npm run start` script that does this automatically

Reply:
added that command after generation https://github.com/aleph-im/aleph-indexer-library/pull/18/commits/b694a7b0a198fa03aa2e68d92f45948241e65553, also the start script on package.json genetated